edb/schema/objects.py

```diff
@@ -2553,15 +2583,57 @@ def delta_property(
         delta: sd.Command,
         fname: str,
         value: Any,
+        orig_value: Any = None,
     ) -> None:
         assert isinstance(scls, InheritingObject)
         inherited_fields = scls.get_inherited_fields(schema)
         delta.set_attribute_value(
             fname,
             value,
+            orig_value=orig_value,
             inherited=fname in inherited_fields,
         )
 
+    @classmethod
+    def compare_obj_field_value(
+        cls: Type[Object_T],
+        field: Field[Type[T]],
+        ours: Optional[Object_T],
+        theirs: Optional[Object_T],
+        *,
+        our_schema: s_schema.Schema,
+        their_schema: s_schema.Schema,
+        context: ComparisonContext,
+        explicit: bool = False,
+    ) -> float:
+        fname = field.name
+        if explicit:
+            our_value = ours.get_explicit_field_value(our_schema, fname, None)
+            their_value = theirs.get_explicit_field_value(their_schema, fname, None)
+        else:
+            our_value = ours.get_field_value(our_schema, fname)
+            their_value = theirs.get_field_value(their_schema, fname)
+
+        similarity = cls._compare_field_value(
+            field,
+            our_value,
+            their_value,
+            our_schema=our_schema,
+            their_schema=their_schema,
+            context=context,
+        )
+
+        # Check to see if this field's inherited status has changed.
+        # If so, this is defintely a change.
+        our_ifs = ours.get_inherited_fields(our_schema)
+        their_ifs = theirs.get_inherited_fields(their_schema)
+
+        if (fname in our_ifs) != (fname in their_ifs):
+            # The change in inherited status decreases the similarity.
+            similarity *= 0.5
```

Comment:
I would set a higher coef here.  Generally if you just want something to register as "altered" you simply need any value < 1, but you should be careful not to tip the overall coefficient product to below 0.6 where it would no longer be considered an "alter".